src/pages/ReviewPage.js

```diff
@@ -25,47 +25,37 @@ const spacingButton = css`
   position: relative;
   top: 2px;
 `
-
 class ReviewPage extends React.Component {
   constructor(props) {
     super(props)
     this.state = { sending: false }
     this.handleSubmit = this.handleSubmit.bind(this)
     this.deleteTempAppointment = this.deleteTempAppointment.bind(this)
   }
-
   handleSubmit() {
     this.setState({ sending: true })
   }
-
   translateReason(reason) {
     if (reason && reason === 'yes') {
       return <Trans>Yes</Trans>
     } else {
       return <Trans>No</Trans>
     }
   }
-
   // from: stackoverflow 'generate a hash from string...'
-  hashFromData(email, paperFileNumber) {
-    var hash = 0,
-      i,
-      chr
-    const keys = email + paperFileNumber
-    if (keys.length === 0) return hash
-    for (i = 0; i < keys.length; i++) {
-      chr = keys.charCodeAt(i)
-      hash = (hash << 5) - hash + chr
-      hash |= 0
-    }
-    return hash
-  }
-
-  deleteTempAppointment() {
-    let tempAppointment = this.props.context.store.calendar.tempAppointment
-    return axios.delete(`/appointments/temp/delete/${tempAppointment._id}`)
-  }
-
+  // hashFromData(email, paperFileNumber) {
+  //   var hash = 0,
+  //     i,
+  //     chr
+  //   const keys = email + paperFileNumber
+  //   if (keys.length === 0) return hash
+  //   for (i = 0; i < keys.length; i++) {
+  //     chr = keys.charCodeAt(i)
+  //     hash = (hash << 5) - hash + chr
+  //     hash |= 0
+  //   }
+  //   return hash
+  // }
```

Comment:
Same as above, keeping this should be unnecessary.

Reply:
Comments have been removed